### ISSUE TITLE
WTF is EBS?

### ISSUE THREAD

**Issue by jlinford** [ParaToolsInc/taucmdr#260]
I know what EBS is ;-) .  But nobody outside of the TAU community does.  Instead of 
```
  --ebs-unwind [depth]        Maximum depth for EBS unwinding.
```
how about
```
--unwind-depth :  Record callstack to specified depth.
```

Since partial arguments are allowed, you could specify `--unwind` and get a sensible default, e.g. 10.  And, when unwinding is used for things other than sampling we won't have to add more flags.

Calling this a bug because it breaks usability.

**Comment by nchaimov** [comment]
Pending successful completion of the test suites, this is fixed as of 26623fa.

I wonder if we should document somewhere what the underlying TAU options are. This feature was added in response to a user asking specifically how to get TAU Commander to set TAU_EBS_UNWIND.

**Comment by jlinford** [comment]
Underlying features should not be documented because:

- Which feature's are activated depend on many factors including OS, Arch, MPI, compiler, etc.  A single taucmdr flag could activate multiple TAU features, or none, depending on the selected experiment.
- If you're trying to activate a specific TAU feature via TAU Commander then you're using the tool backwards.  The correct usage is to tell the tool what you want then sit back and let the tool do whatever it needs to bring that result to you.
- If you understand TAU internals then it might be faster for you to just use TAU directly.  Or not ;-)
- Given the slap-dab TAU development process, maintaining that documentation would be a nightmare.  It's virtually guaranteed to be out of date within months.
